src/scipp/plotting/toolbar.py

```diff
@@ -272,6 +273,14 @@ def __init__(self, *args, **kwargs):
                         description="Z",
                         tooltip="Camera to Z normal. "
                         "Click twice to flip the view direction.")
+        self.add_togglebutton(name="toggle_axes_helper",
+                              value=True,
+                              icon="bar-chart",
```

Comment:
I can't draw the connection between the icon and what the button does?

Reply:
Yeah, it was quite difficult to find an icon.
Other ideas included: &#x1F7C0; and &#x27C0;
Maybe they are better? Unfortunately, when using unicode characters, they are smaller than the icons, and it doesn't look quite as unified. But in the future, there should be a way to control font size and weight in the widgets (changes not released yet, but you can see here: https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20Styling.html#The-style-attribute

Reply:
The arrow-like almost looks like 3d axes?

Reply:
I changed it to 
![Screenshot at 2021-11-08 13-55-38](https://user-images.githubusercontent.com/39047984/140745852-a2fcee2c-0e00-4b8b-90f1-36087ffd1e91.png)
See what you think

Reply:
Looks ok?